chart/kubeapps/README.md

```diff
@@ -34,19 +44,26 @@ It also packages the [Bitnami MongoDB chart](https://github.com/helm/charts/tree
 
 To install the chart with the release name `kubeapps`:
 
+For Helm v2:
+
 ```console
 $ helm repo add bitnami https://charts.bitnami.com/bitnami
 $ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
 ```
 
-> **IMPORTANT** This assumes an insecure Helm installation, which is not recommended in production. See [the documentation to learn how to secure Helm and Kubeapps in production](https://github.com/kubeapps/kubeapps/blob/master/docs/user/securing-kubeapps.md).
+> **IMPORTANT** This assumes an insecure Helm v2 installation, which is not recommended in production. See [the documentation to learn how to secure Helm v2 and Kubeapps in production](https://github.com/kubeapps/kubeapps/blob/master/docs/user/securing-kubeapps.md).
+
+For Helm v3:
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm install kubeapps --namespace kubeapps bitnami/kubeapps --set useHelm3=true
+```
 
 The command deploys Kubeapps on the Kubernetes cluster in the `kubeapps` namespace. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
 
 > **Caveat**: Only one Kubeapps installation is supported per namespace
 
-> **Tip**: List all releases using `helm list`
-
```

Comment:
Why did you remove this? If we add the `--all-namespaces` flag, it still lists releases across namespaces (or the `--all` flag for all statuses).

Reply:
I removed it because the command is different for helm 2 and helm 3 and I didn't want to add more clutter to the comment (also the tip is not helping much)